strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/BinaryConfigurationServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package org.carlspring.strongbox.services.impl;
+
+import org.carlspring.strongbox.configuration.BinaryConfiguration;
+import org.carlspring.strongbox.data.service.CommonCrudService;
+import org.carlspring.strongbox.services.BinaryConfigurationService;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Transactional
+@Service
+public class BinaryConfigurationServiceImpl
```

Comment:
I thought the configuration was stored as objects. From I am seeing, it appears that the XML is being stored in the DB? Is this the best approach? (Just checking what you think).

Reply:
I don't see a better choice right now, until we fix SB-930.